Sources/Purchasing/Purchases/Attribution.swift

```diff
@@ -40,6 +47,34 @@ import Foundation
 
 }
 
+public extension Attribution {
+
+    /**
+     * Enable automatic collection of AdServices attribution token.
+     *
+     * Should match OS availability in https://developer.apple.com/documentation/ad_services
+     */
+    @available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *)
+    @available(tvOS, unavailable)
+    @available(watchOS, unavailable)
+    @objc func enableAdServicesAttributionTokenCollection() {
+        self.automaticAdServicesAttributionTokenCollection = true
+        self.postAdServicesTokenIfNeeded()
```

Comment:
This is only called after setting it to true, so it doesn't need the "if needed" and guard.

Reply:
oh also -- the "ifNeeded" kinda covers checking whether it's already been posted, i.e. if a token exists in UserDefaults

Reply:
Oh I missed that! Never mind then.